packages/host/app/templates/application.gts

```diff
@@ -12,6 +12,7 @@ class ApplicationRouteComponent extends Component<ApplicationRouteSignature> {
   <template>
     {{outlet}}
     <CardPrerender />
+    <div data-test-boxel-root style='display: none;'></div>
```

Comment:
This is a signal that we use to know that the ember app has loaded for the Realm DOM tests